frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -282,7 +385,7 @@ export class ImportWizard extends Wizard {
     await importWizard.waitForCreation();
 
     // Navigate to detail page
-    await importWizard.navigateToDetail();
+    // await importWizard.navigateToDetail();
```

Comment:
Why commented?

Reply:
Now this is not relevant, I've splitted that part for RHV and VMWare and no comment there